fvm/fvm_test.go

```diff
@@ -241,7 +241,7 @@ func TestPrograms(t *testing.T) {
 	)
 
 	t.Run("script execution programs are not committed",
-		newVMTest().run(
+		newVMTest().withBootstrapProcedureOptions(fvm.WithExecutionMemoryLimit(math.MaxUint64)).run(
```

Comment:
This shouldn't be needed in this unit test. 
We set this as an option, so max should be the default not to change the existing behaviour.

Reply:
This is not necessary: [389fb3c](https://github.com/onflow/flow-go/pull/2474/commits/389fb3cc0adbb7727024a2e00e337500c1994894)